docs/user/TLS_PASSTHROUGH.md

```diff
@@ -0,0 +1,118 @@
+## TLS Passthrough
+This guide will walk through the steps required to configure TLS Passthrough via Envoy Gateway. Unlike configuring Secure Gateways, where the Gateway terminates the client TLS connection, TLS Passthrough allows the application itself to terminate the TLS connection, while the Gateway routes the requests to the application based on SNI headers.
+
+
+### Prerequisites
+- A Kubernetes cluster with `kubectl` context configured for the cluster.
+- OpenSSL to generate TLS assets.
+
+__Note:__ Envoy Gateway is tested against Kubernetes v1.24.0.
+
+### Installation
+Follow the steps from the [Quickstart Guide](QUICKSTART.md) to install Envoy Gateway and the example manifest.
+Before proceeding, you should be able to curl the example backend using HTTP.
+
+### TLS Certificates
+
+Generate the certificates and keys used by the Service to terminate client TLS connections. 
+For the application, we'll deploy a sample nginx app, with the certificates loaded in the application Pod.
+
+__Note:__ These certificates will not be used by the Gateway, but will remain in the application scope.
+
+For macOS users, verify curl is compiled with the LibreSSL library:
+```shell
+curl --version | grep LibreSSL
+curl 7.54.0 (x86_64-apple-darwin17.0) libcurl/7.54.0 LibreSSL/2.0.20 zlib/1.2.11 nghttp2/1.24.0
+```
+
+Create a root certificate and private key to sign certificates:
+```shell
+openssl req -x509 -sha256 -nodes -days 365 -newkey rsa:2048 -subj '/O=example Inc./CN=example.com' -keyout example.com.key -out example.com.crt
+```
+
+Create a certificate and a private key for `passthrough.example.com`:
+```shell
+openssl req -out passthrough.example.com.csr -newkey rsa:2048 -nodes -keyout passthrough.example.com.key -subj "/CN=passthrough.example.com/O=some organization"
+openssl x509 -req -sha256 -days 365 -CA example.com.crt -CAkey example.com.key -set_serial 0 -in passthrough.example.com.csr -out passthrough.example.com.crt
+```
+
+Store the cert/keys in A Secret:
+```shell
+kubectl create secret tls nginx-server-certs --key=passthrough.example.com.key --cert=passthrough.example.com.crt
+```
+
+### Application config
+
+Create a file for the nginx server:
+```shell
+cat <<\EOF > ./nginx-passthrough.conf
```

Comment:
The plan is to get rid of httpbin and consolidate all examples to use the same kube conformance echo server. The echo server [supports TLS](https://github.com/kubernetes-sigs/ingress-controller-conformance/blob/master/images/echoserver/echoserver.go#L108-L123). Refer to https://github.com/envoyproxy/gateway/pull/558/ for additional details regarding the echo server.